drivers/usb/device/Kconfig

```diff
@@ -63,7 +63,7 @@ config USB_DC_STM32
 config USB_DC_STM32_CLOCK_CHECK
 	bool "Runtime USB 48MHz clock check"
 	depends on USB_DC_STM32
-	default y if !(SOC_SERIES_STM32F1X || SOC_SERIES_STM32F3X || SOC_SERIES_STM32U5X)
+	default y if !(SOC_SERIES_STM32F1X || SOC_SERIES_STM32F3X || SOC_SERIES_STM32U5X || SOC_SERIES_STM32H7X || SOC_SERIES_STM32H7RSX)
```

Comment:
See previous comment about modifying original commits.
Also, doing changes on SOC_SERIES_STM32H7X should be done using a dedicated commit and proper justification. In this case a dedicated PR would even be required